pysages/methods/spline_string.py

```diff
@@ -0,0 +1,297 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020-2021: PySAGES contributors
+# See LICENSE.md and CONTRIBUTORS.md at https://github.com/SSAGESLabs/PySAGES
+
+"""
+Spline String method aka Improved String method.
+
+The improved string methods finds the local minimum free-energy path (MFEP) trough a given landscape.
+It uses umbrella integration to advance the discretized replicas in the CV space.
+A spline interpolation helps to (equally) space the CV points along the string.
+The process converges if the MFEP is found.
+The final free-energy profile is calculated the same way as in UmbrellaIntegration.
+We aim to implement this: `Weinan, E., et. al. J. Chem. Phys. 126.16 (2007): 164103 <https://www.researchgate.net/profile/Eric-Vanden-Eijnden/publication/6351826_Simplified_and_Improved_String_Method_for_Computing_the_Minimum_Energy_Paths_in_Barrier_Crossing_Events/links/5523eeab0cf2c815e073df2f/Simplified-and-Improved-String-Method-for-Computing-the-Minimum-Energy-Paths-in-Barrier-Crossing-Events.pdf>`_.
```

Comment:
```suggestion
We aim to implement this: `Weinan, E., et. al. J. Chem. Phys. 126.16 (2007): 164103 <https://doi.org/10.1063/1.2720838>`_.
```

or did you wanted to provide a direct link to the PDF for everyone to access? I think Google Scholar provides this link as well, so I'm no sure what to use here.